Client/JsonLDClient.php

```diff
@@ -109,10 +110,10 @@ public function delete(object $object, array $params = []): void
      * @template T
      * @psalm-param class-string<T> $className
      * @param array $params
-     * @return ApiIterable<T>
+     * @return \Iterator<T>
      * @throws JsonLDException
      */
-    public function getMany(string $className, array $params): ApiIterable
+    public function getMany(string $className, array $params, bool $useCache = false): \Iterator
```

Comment:
I'm not sure I see the benefit of this? On the contrary actually. Can you explain why this is done?

Reply:
The flag to not use the cache? Two reasons: one, to match getById and getSingleton (which also use caching); two, to allow us to have the old behaviour if we need it (which we probably will, there are things that you need to be sure are up-to-date.

If you mean the return type-hint, it's mostly to make testing easier. The fact that it's an ApiIterable is really an implementation detail, all the consumer should care about is that they can iterate it.

Reply:
Fair enough, thanks. Could you add the new @param to the docblock please?